data/plugins.json

```diff
@@ -20,11 +20,11 @@
     "node": 6
   },
   "transform-es2015-classes": {
-    "chrome": 46,
+    "chrome": 45,
     "edge": 13,
     "firefox": 45,
     "safari": 10,
-    "node": 5
+    "node": 4
```

Comment:
I wrote, a long time ago in phabricator (no idea on the bug number), that transform-rest-spread should, at least, detect if super with spread is being used, and bail if transform-es2015-classes is not also being used.